### PR TITLE
Malabranch (malayim review needed)

### PR DESCRIPTION
**Pull request by Bib711**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Bib711** [comment]
Translated HOWTO.md to MALAYALAM

**Comment by eshellman** [comment]
also need to add link in README

**Comment by Bib711** [comment]
What link?

**Comment by eshellman** [comment]
link to the adde file

**Review by gaurovgiri — APPROVED** [review]

File: docs/HOWTO-ml.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+• [പുൾ അഭ്യർത്ഥനകളെക്കുറിച്ച്](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+• [ഒരു പുൾ അഭ്യർത്ഥന സൃഷ്ടിക്കുന്നു](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+• [GitHub ഹലോ വേൾഡ്](https://docs.github.com/en/get-started/quickstart/hello-world)
+• [തുടക്കക്കാർക്കുള്ള YouTube - GitHub ട്യൂട്ടോറിയൽ](https://www.youtube.com/watch?v=0fKg7e37bQE)
+• [YouTube - എങ്ങനെ ഒരു GitHub റിപ്പോ ഫോർക്ക് ചെയ്യുകയും ഒരു പുൾ അഭ്യർത്ഥന സമർപ്പിക്കുകയും ചെയ്യാം](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+• [YouTube - മാർക്ക്ഡൗൺ ക്രാഷ് കോഴ്സ്](https://www.youtube.com/watch?v=HUBNt18RFbo)
+
+ചോദ്യങ്ങൾ ചോദിക്കാൻ മടിക്കരുത്; എല്ലാ സംഭാവകരും ഒരു ആദ്യ PR-ൽ ആരംഭിച്ചു. അതിനാൽ... എന്തുകൊണ്ട് നമ്മുടെ [വലിയ, വളരുന്ന](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) സമൂഹത്തിൽ ചേരുന്നില്ല.
```
Comment (rashivkp) [inline docs/HOWTO-ml.md:18]:
```suggestion
ചോദ്യങ്ങൾ ചോദിക്കാൻ മടിക്കരുത്; എല്ലാ സംഭാവകരും തുടങ്ങിയത് ഒരു ആദ്യ PR കൊണ്ടാണ്. അതിനാൽ... എന്തുകൊണ്ട് നമ്മുടെ [വലിയ, വളരുന്ന](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) സമൂഹത്തിൽ ചേരുന്നില്ല.
```

File: docs/HOWTO-ml.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+[![EbookFoundation/free-programming-books's Monthly Active Contributors graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)
+
+</details>
+
+നിങ്ങൾ അനുഭവപരിചയമുള്ള ഒരു ഓപ്പൺ സോഴ്‌സ് സംഭാവകനാണെങ്കിൽ പോലും, നിങ്ങളെ ഞെരുക്കിയേക്കാവുന്ന ചില കാര്യങ്ങളുണ്ട്. നിങ്ങളുടെ പിആർ സമർപ്പിച്ചുകഴിഞ്ഞാൽ,***GitHub പ്രവർത്തനങ്ങൾ* ഒരു *ലിന്റർ* പ്രവർത്തിപ്പിക്കും, ഇടയ്ക്കിടെ സ്പെയ്സിംഗിലോ അക്ഷരമാലാക്രമത്തിലോ ചെറിയ പ്രശ്നങ്ങൾ കണ്ടെത്തും**. നിങ്ങൾക്ക് ഒരു പച്ച ബട്ടൺ ലഭിക്കുകയാണെങ്കിൽ, എല്ലാം അവലോകനത്തിന് തയ്യാറാണ്; ഇല്ലെങ്കിൽ, ചെക്കിന് താഴെയുള്ള "വിശദാംശങ്ങൾ" ക്ലിക്ക് ചെയ്യുക, അത് ലിന്ററിന് ഇഷ്ടപ്പെടാത്തത് എന്താണെന്ന് കണ്ടെത്താനായില്ല, കൂടാതെ നിങ്ങളുടെ പിആർ തുറന്ന ബ്രാഞ്ചിലേക്ക് ഒരു പുതിയ പ്രതിബദ്ധത ചേർക്കുന്നതിൽ പ്രശ്നം പരിഹരിക്കുക.
```
Comment (rashivkp) [inline docs/HOWTO-ml.md:29]:
```suggestion
നിങ്ങൾ അനുഭവപരിചയമുള്ള ഒരു ഓപ്പൺ സോഴ്‌സ് സംഭാവകനാണെങ്കിൽ പോലും, ചില കാര്യങ്ങള്‍ നിങ്ങളെ പ്രയാസപ്പെടുത്തിയേക്കാം. നിങ്ങളുടെ പിആർ സമർപ്പിച്ചുകഴിഞ്ഞാൽ,***GitHub Actions* ഒരു *ലിന്റർ* പ്രവർത്തിപ്പിക്കും, പലപ്പോഴും സ്പെയ്സിംഗിലോ അക്ഷരമാലാക്രമത്തിലോ ചെറിയ പ്രശ്നങ്ങൾ കണ്ടെത്തും**. നിങ്ങൾക്ക് ഒരു പച്ച ബട്ടൺ ലഭിക്കുകയാണെങ്കിൽ, എല്ലാം അവലോകനത്തിന് തയ്യാറാണ്; ഇല്ലെങ്കിൽ, ലിന്ററിന് ഇഷ്ടപ്പെടാത്തത് എന്താണെന്ന്  മനസ്സിലാക്കാന്‍ ചെക്കിന് താഴെയുള്ള "വിശദാംശങ്ങൾ" ക്ലിക്ക് ചെയ്യുക, കൂടാതെ നിങ്ങളുടെ പിആർ തുറന്ന ബ്രാഞ്ചിലേക്ക് ഒരു പുതിയ commit ചേര്‍ത്ത് പ്രശ്നം പരിഹരിക്കുക.
```

File: docs/HOWTO-ml.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+**'സൗജന്യ-പ്രോഗ്രാമിംഗ്-ബുക്കുകളിലേക്ക്' സ്വാഗതം!**
+
+പുതിയ സംഭാവകരെ ഞങ്ങൾ സ്വാഗതം ചെയ്യുന്നു; GitHub-ൽ അവരുടെ ആദ്യത്തെ പുൾ അഭ്യർത്ഥന (പിആർ) നടത്തുന്നവർ പോലും. നിങ്ങൾ അത്തരത്തിലൊരാളാണെങ്കിൽ, സഹായിച്ചേക്കാവുന്ന ചില ഉറവിടങ്ങൾ ഇതാ:
+
+• [പുൾ അഭ്യർത്ഥനകളെക്കുറിച്ച്](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+• [ഒരു പുൾ അഭ്യർത്ഥന സൃഷ്ടിക്കുന്നു](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+• [GitHub ഹലോ വേൾഡ്](https://docs.github.com/en/get-started/quickstart/hello-world)
+• [തുടക്കക്കാർക്കുള്ള YouTube - GitHub ട്യൂട്ടോറിയൽ](https://www.youtube.com/watch?v=0fKg7e37bQE)
+• [YouTube - എങ്ങനെ ഒരു GitHub റിപ്പോ ഫോർക്ക് ചെയ്യുകയും ഒരു പുൾ അഭ്യർത്ഥന സമർപ്പിക്കുകയും ചെയ്യാം](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+• [YouTube - മാർക്ക്ഡൗൺ ക്രാഷ് കോഴ്സ്](https://www.youtube.com/watch?v=HUBNt18RFbo)
```
Comment (rashivkp) [inline docs/HOWTO-ml.md:16]:
```suggestion
• [YouTube - Markdown ക്രാഷ് കോഴ്സ്](https://www.youtube.com/watch?v=HUBNt18RFbo)
```

**Review by rashivkp — CHANGES_REQUESTED** [review]
PR seems like word by word translation. :smile:   Anyway trying to improve it a bit.

**Comment by Bib711** [comment]
Ok

**Comment by Vaporjawn** [comment]
@eshellman I've addressed the comments with [this PR](https://github.com/EbookFoundation/free-programming-books/pull/11000), if you'd like to move along with this instead

**Comment by felixdiria** [comment]
Diff

**Comment by eshellman** [comment]
closed following #11000

**Pull request closed without merging**